.github/workflows/SwiftFormat.yml

```diff
@@ -56,8 +58,8 @@ jobs:
         with:
           draft: true
           prerelease: true
-          release_name: SwiftFormat-${{ matrix.tag }}
-          tag_name: SwiftFormat-${{ matrix.tag }}
+          release_name: SwiftFormat-${{ matrix.tag }}-${{ matrix.swift_tag }}
```

Comment:
The release name will be:
`SwiftFormat-0.51.10-DEVELOPMENT-SNAPSHOT-2023-05-22-a`

Which can be a bit confusing to parse. What about:
`SwiftFormat-${{ matrix.tag }}_for_swift_toolchain_${{ matrix.swift_tag }}`

Which would become, for example:
`SwiftFormat-0.51.10_for_swift_toolchain_DEVELOPMENT-SNAPSHOT-2023-05-22-a`

Reply:
Can we do `SwiftFormat-0.51.10 (Swift 5.8)` or `SwiftFormat-0.51.10 (Swift development)`?

Reply:
> Can we do `SwiftFormat-0.51.10 (Swift 5.8)` or `SwiftFormat-0.51.10 (Swift development)`?

Yes! I think the release name can have parenthesis and spaces, if so, it is much better :-)